src/afl-fuzz-stats.c

```diff
@@ -632,7 +630,7 @@ void show_stats_normal(afl_state_t *afl) {
 
   cur_ms = get_cur_time();
 
-  if (afl->most_time_key) {
+  if (afl->most_time_key && afl->queue_cycle) {
```

Comment:
queue_cycle is 0 on startup and only increases to 1 once a cycle is done.
this cannot be right.

Reply:
I thought it was increased to 1 right as the first cycle is started, and only when PRINTING we subtract one. to make the first cycle look like 0 too... let me double check.

Reply:
https://github.com/AFLplusplus/AFLplusplus/blob/stable/src/afl-fuzz-stats.c#L379 i.e. queue cycle should be 0 while loading, and 1 as soon as we are in the first cycle - it's just printed with an offset of 1.

Reply:
du kannst ja mal mit -V10 starten und einer queue mit inhalt und einem echten target ... der sollte dann nicht nach 10 sekunden aussteigen wie er sollte.

Reply:
ok question: What's the expected semantic for a real queue (that takes, say 30 seconds to load) and running with -V 0 and -V 10. I am expecting -V 0 to load the queue for 30 seconds and then terminate right away, and similarly -V 10 should load the queue for 30 seconds + do 10 seconds of fuzzing? 
Maybe our tests didn't catch that because the test target cycles within < 1 second. Let me double check.

Reply:
yes that is my expectation. also that syncing, calibration etc. happen, and V10 still would do 10 seconds of fuzzing after that

Reply:
let me check if that's what's happening & if not fix it again :P

Reply:
validated that both: 
```
make; AFL_IMPORT_FIRST=1 AFL_NO_UI=1 ./afl-fuzz -i /tmp/in -o /tmp/syncdir -M fuzzer -V0 ./test-target/libjpeg-turbo/build/djpeg
```

```
make; AFL_IMPORT_FIRST=1 AFL_NO_UI=1 ./afl-fuzz -i /tmp/in -o /tmp/syncdir -M fuzzer -V10 ./test-target/libjpeg-turbo/build/djpeg
```
do the right thing